### PR TITLE
✨DCS-1281: Add link to confirm arrival journey for arriving from custody suite & 📦 update dependencies

### PR DESCRIPTION
**Pull request by connormaglynn**

### CODE REVIEW

File: integration_tests/integration/choosePrisoner.spec.ts [hunk 1]
```diff
@@ -134,26 +146,18 @@ context('Choose Prisoner', () => {
       })
   })
 
-  it('Only court arrivals with no current booking will have a link leading to the Confirm arrival page', () => {
+  it.only('Only court arrivals with no current booking and arrivals from custody suites will have a link leading to the Confirm arrival page', () => {
```
Comment (andrewrlee) [inline integration_tests/integration/choosePrisoner.spec.ts:149]: think you need to remove the `only`

File: integration_tests/mockApis/responses/expectedArrivals.ts [hunk 2]
```diff
@@ -20,6 +21,7 @@ export default [
     fromLocationType: 'CUSTODY_SUITE',
   },
   {
+    id: '00000-333333',
```
Comment (andrewrlee) [inline integration_tests/mockApis/responses/expectedArrivals.ts:24]: each arrival should have a unique id?

File: integration_tests/integration/choosePrisoner.spec.ts [hunk 3]
```diff
@@ -169,4 +173,16 @@ context('Choose Prisoner', () => {
     choosePrisonerPage.arrivalFrom('COURT')(2).confirm().should('not.exist')
     choosePrisonerPage.arrivalFrom('COURT')(3).confirm().should('not.exist')
   })
+
+  function confirmLinkIsClickableFor(
+    choosePrisonerPage: ChoosePrisonerPage,
+    stub: Record<string, string | boolean>,
+    arrivalType: 'COURT' | 'PRISON' | 'CUSTODY_SUITE',
+    rowNumber = 1
```
Comment (andrewrlee) [inline integration_tests/integration/choosePrisoner.spec.ts:181]: probably better to not default if the default only works for 1/4 situations

File: integration_tests/integration/choosePrisoner.spec.ts [hunk 4]
```diff
@@ -134,26 +146,18 @@ context('Choose Prisoner', () => {
...
+    confirmLinkIsClickableFor(choosePrisonerPage, expectedArrivalMatchedInNomisWithNoBooking, 'COURT', 2)
+    confirmLinkIsClickableFor(choosePrisonerPage, expectedArrivalUnmatchedInNomis, 'COURT', 3)
 
-    cy.go('back')
-
-    cy.task('stubExpectedArrival', expectedArrivalUnmatchedInNomis)
-    choosePrisonerPage.arrivalFrom('COURT')(3).confirm().should('exist').click()
-    Page.verifyOnPage(ConfirmArrivalPage).prisonNumber().should('not.exist')
+    confirmLinkIsClickableFor(choosePrisonerPage, expectedArrivalFromCustodySuite, 'CUSTODY_SUITE')
+    confirmLinkIsClickableFor(choosePrisonerPage, expectedArrivalFromCustodySuite, 'CUSTODY_SUITE', 2)
```
Comment (andrewrlee) [inline integration_tests/integration/choosePrisoner.spec.ts:160]: meh.. why not each row in turn?

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**